tools/windows-node-installer/test/e2e/azure_test.go

```diff
@@ -0,0 +1,197 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"reflect"
+	"regexp"
+	"strings"
+	"testing"
+
+	"github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2019-03-01/compute"
+	"github.com/Azure/azure-sdk-for-go/services/network/mgmt/2019-04-01/network"
+	"github.com/Azure/go-autorest/autorest/azure"
+	"github.com/Azure/go-autorest/autorest/azure/auth"
+	"github.com/openshift/windows-machine-config-operator/tools/windows-node-installer/pkg/client"
+	"github.com/openshift/windows-machine-config-operator/tools/windows-node-installer/pkg/resource"
+)
+
+const (
+	// winRm Https port for the Windows node.
+	winRmPortHttps = "5986"
```

Comment:
I think we've been using `winRM` till now.

Reply:
Please `winRM`

Reply:
>I will move that to the types package so that the const can be used across the cloud providers.

This should be done in a separate commit